sdk/src/main/scala/com/horizen/account/state/AccountState.scala

```diff
@@ -380,6 +383,30 @@ class AccountState(val params: NetworkParams,
     res
   }
 
+  override def validate(tx: SidechainTypes#SCAT): Try[Unit] = Try {
+    tx.semanticValidity()
+
+    if (!tx.isInstanceOf[MC2SCAggregatedTransaction]) {
```

Comment:
Why not checking directly if it is a EthereumTransaction instead of checking if it is not an MC2SCAggregatedTransaction?

Reply:
good point,
must be `EthereumTransaction` here